src/cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -1216,6 +1224,75 @@ msgstr "Nicht festgelegt"
 msgid "Edit event"
 msgstr "Veranstaltung bearbeiten"
 
+#: templates/feedback/admin_feedback_list.html:10
+#: templates/feedback/region_feedback_list.html:10
+#: templates/feedback/region_feedback_list.html:112
+msgid "Type"
+msgstr "Typ"
+
+#: templates/feedback/admin_feedback_list.html:11
+#: templates/feedback/region_feedback_list.html:11
+msgid "Object"
+msgstr "Objekt"
+
+#: templates/feedback/admin_feedback_list.html:12
+#: templates/feedback/region_feedback_list.html:12
+msgid "Comment"
+msgstr "Kommentar"
+
+#: templates/feedback/admin_feedback_list.html:13
+#: templates/feedback/region_feedback_list.html:13
+#: templates/feedback/region_feedback_list.html:108
+msgid "Rating"
+msgstr "Bewertung"
+
+#: templates/feedback/admin_feedback_list.html:19
+msgid "Page"
+msgstr "Seite"
+
+#: templates/feedback/admin_feedback_list.html:32
+#: templates/feedback/region_feedback_list.html:28
+msgid "Event"
+msgstr "Veranstaltungen"
```

Comment:
"Event" is singular, so it should be "Veranstaltung" instead of "Veranstaltungen".